ui/app/models/aws/root-config.js

```diff
@@ -45,40 +45,50 @@ export default class AwsRootConfig extends Model {
   iamEndpoint;
   @attr('string', { label: 'STS endpoint' }) stsEndpoint;
   @attr('number', {
-    label: 'Maximum retries',
     subText: 'Number of max retries the client should use for recoverable errors. Default is -1.',
   })
   maxRetries;
 
+  configurableParams = [
+    'roleArn',
+    'identityTokenAudience',
+    'identityTokenTtl',
+    'accessKey',
+    'secretKey',
+    'region',
+    'iamEndpoint',
+    'stsEndpoint',
+    'maxRetries',
+  ];
+
+  get isWifPluginConfigured() {
+    return !!this.identityTokenAudience || !!this.identityTokenTtl || !!this.roleArn;
+  }
+
+  get isAccountPluginConfigured() {
+    return !!this.accessKey;
+  }
+
   get displayAttrs() {
```

Comment:
this looks like I added `secretKey` to something I shouldn't have. It's a little hard to parse, but the views are still the same. For `displayAttrs` I filter out secret-key.